util/src/Digest.cpp

```diff
@@ -98,8 +98,9 @@ Digest::Digest(char* buf, size_t len) { DigestUtil::compute(buf, len, (char*)d,
 
 std::string Digest::toString() const {
   std::ostringstream oss;
-  for (size_t i = 0; i < DigestType::DIGESTSIZE; ++i)
+  for (size_t i = 0; i < DigestType::DIGESTSIZE; ++i) {
     oss << std::setfill('0') << std::setw(2) << std::hex << std::uppercase << (0xff & (unsigned int)d[i]);
```

Comment:
should we use the apis from hextools to print the digest in hexa decimal format.

Reply:
`toString()` replaced with hex tool api in [this commit](https://github.com/vmware/concord-bft/pull/2457/commits/187c7eb432d9e177dce8488a23aa554ead109916).